internal/builders/nodejs/README.md

```diff
@@ -190,22 +190,27 @@ name information is tracked on [issue #372](https://github.com/slsa-framework/sl
 
 ### Supported Triggers
 
-The following [GitHub trigger
-events](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows)
-are fully supported and tested:
-
-- `schedule`
-- `push` (including new tags)
-- `release`
-- Manual run via `workflow_dispatch`
-
-However, in practice, most triggers should work with the exception of
-`pull_request`. If you would like support for `pull_request`, please tell us
-about your use case on
+Only the following [event types] are supported:
+
+| Supported event type  | Event description                          |
+| --------------------- | ------------------------------------------ |
```

Comment:
want to update the the slsa-verifier Action code and remove the deployement https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/actions/verify-token/src/validate.ts#L32-L38?

Reply:
Will submit a separate PR